app/src/main/java/org/simple/clinic/settings/SettingsInit.kt

```diff
@@ -8,8 +8,8 @@ class SettingsInit : Init<SettingsModel, SettingsEffect> {
 
   override fun init(model: SettingsModel): First<SettingsModel, SettingsEffect> {
     val effects = when {
-      model.userDetailsQueried -> setOf(LoadCurrentLanguageEffect)
-      else -> setOf(LoadUserDetailsEffect, LoadCurrentLanguageEffect)
+      model.userDetailsQueried -> setOf(LoadCurrentLanguageEffect, LoadAppVersionEffect(model.applicationId))
```

Comment:
The reason this is done is in order to prevent querying the app version again on state restoration unless we expect the data to change. Since the data won't change, should we add a check to prevent the app version from being fetched again as well?